### PR TITLE
docs: update README

### PR DESCRIPTION
**Pull request by whentojump**
As I checked and ran the code blocks in README I propose these changes.

Will create another PR regarding the text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,25 +108,28 @@ Please proceed to the [Kick-the-tire Instructions](#3-kick-the-tire-instructions
...
-Deleting cluster "acto-0-cluster-0" ...                                                                                                                                                                             
-Creating a Kind cluster...                                                                                                                                                                                            
-Deploying operator...                                                                                                                                                                                               
-Operator deployed                                                                                                                             
-Bug rdoptwo-287 reproduced!                                                                                                                                                                                         
+```text
+Reproducing single bug rdoptwo-287 in OCK-RedisOp!
+Preparing required images...
+Deleting cluster "acto-0-cluster-0" ...
+Creating kind cluster...
```
Comment (whentojump) [inline README.md:129]: Note: results are slightly different in text. Looks mine is consistent with the latest code of `sosp-ae`

File: README.md [hunk 2]
```diff
@@ -136,9 +139,10 @@ Bug category: undesired_state
 
 To reproduce the 56 bugs in Table 5, please execute the tests by running:
 
-```
+```sh
+cd ~/workdir/acto/
 make
-python3 reproduce_bugs.py -n NUM_WORKERS
+python3 reproduce_bugs.py -n <NUM_WORKERS>
```
Comment (whentojump) [inline README.md:145]: Note: my intention is to make it more explicit this is only a placeholder

File: README.md [hunk 3]
```diff
@@ -151,25 +155,26 @@ After it finishes, you will find `table5.txt`, and `table6.txt`, and `table7.txt
...
+CockroachOp                    3               0                 2                   0        5
+KnativeOp                      1               0                 2                   0        3
+OCK-RedisOp                    4               1                 3                   1        9
+OFC-MongoDBOp                  3               1                 2                   2        8
+PCN-MongoDBOp                  4               0                 0                   1        5
+RabbitMQOp                     3               0                 0                   0        3
+SAH-RedisOp                    2               0                 0                   1        3
+TiDBOp                         2               1                 0                   1        4
+XtraDBOp                       4               0                 1                   1        6
+ZookeeperOp                    4               1                 0                   1        6
```
Comment (whentojump) [inline README.md:171]: Note: the results are slightly different as for the first column (operator name), but not the numbers. Looks like mine is consistent with the latest code of `sosp-ae`

File: README.md (hunk 1 above)
Comment (tylergu) [reply README.md:129]: I will update the print statements in Acto to match the current text.

**Comment by tylergu** [comment]
@whentojump Thanks for spotting the differences. I pushed a change for the print statements so that the output of the Kick-the-tire stage matches the current one, other changes look good to me.

File: README.md (hunk 1 above)
Comment (whentojump) [reply README.md]: Reverted relevant changes in this PR

**Pull request merged**